lib/common/icu.net.dll.config

```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0"?>
-<configuration>
-  <dllmap dll="icuin40.dll" target="libicui18n.so.40"/>
-  <dllmap dll="icuuc40.dll" target="libicuuc.so.40"/>
-  <dllmap dll="icuin48.dll" target="libicui18n.so.48"/>
-  <dllmap dll="icuuc48.dll" target="libicuuc.so.48"/>
-  <dllmap dll="icuin50.dll" target="libicui18n.so.50"/>
-  <dllmap dll="icuuc50.dll" target="libicuuc.so.50"/>
-<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.0,Profile=Client"/></startup></configuration>
```

Comment:
Looks like something to support Linux; I'm not clear why we would no longer need it, but I don't anything about how the Linux version ships.